bindgen/assembly/index.ts

```diff
@@ -13,24 +14,24 @@ function isNull<T>(t: T): bool {
   return false;
 }
 
-//@ts-ignore
+// @ts-ignore
 @global
 function notPayable(): void {
   assert(Context.attachedDeposit == u128.Zero, "Method doesn't accept deposit");
 }
 
-//@ts-ignore
+// @ts-ignore
 @global
-function assertNonNull<T>(name: string, t: T): T {
+function requireParameter<T>(name: string): T {
   assert(
-    !isNull<T>(t),
+    false,
```

Comment:
assert false?

Reply:
It's the same as throwing an error.  I can switch to that if you prefer.

Reply:
Oh, I see, this makes sense now. Yes, I think throwing an error would probably read more clearly.